torch_xla/csrc/runtime/ifrt_computation_client.cc

```diff
@@ -78,10 +66,26 @@ torch::lazy::hash_t hash_comp_env(
   hash = torch::lazy::HashCombine(
       hash, torch::lazy::StringHash(platform_version.c_str()));
   // Include global devices in the hash, ensuring order is consistent.
+  xla::ifrt::DeviceList::Devices ifrt_devices;
```

Comment:
Create an extra copy as this would be evicted/destroyed later in the scope. On the other hand, the input `ordered_devices` is passed as reference, and the caller should still be able to use after this call.